<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tableau_bord</class>
 <widget class="QMainWindow" name="Tableau_bord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Restaurant Manager Pro - Tableau de Bord</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QLabel" name="labelDashboardTitle">
      <property name="styleSheet">
       <string>
        font-size: 24px;
        color: #37474f;
        font-weight: bold;
        padding: 20px;
        background-color: white;
        border-radius: 12px;
        margin-bottom: 10px;
       </string>
      </property>
      <property name="text">
       <string>üè™ Tableau de Bord - Restaurant</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutMain">
      <item>
       <widget class="QFrame" name="Salle_restauration">
        <property name="styleSheet">
         <string>
          QFrame {
              background-color: #e3f2fd;
              border-radius: 12px;
              padding: 15px;
          }
         </string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayoutSalle">
         <item>
          <widget class="QLabel" name="labelSalleRestauration">
           <property name="styleSheet">
            <string>
             font-size: 18px;
             font-weight: bold;
             color: #1565c0;
             margin-bottom: 10px;
            </string>
           </property>
           <property name="text">
            <string>üçΩÔ∏è Salle de Restauration</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayoutSalle">
           <item row="0" column="0">
            <widget class="QLabel" name="labelPlacesOccupees">
             <property name="text">
              <string>Places Occup√©es :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLCDNumber" name="lcdPlacesOccupees">
             <property name="styleSheet">
              <string>
               background-color: #1565c0;
               color: white;
               border-radius: 6px;
              </string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QProgressBar" name="progressPlacesOccupees">
             <property name="value">
              <number>50</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="Stock">
        <property name="styleSheet">
         <string>
          QFrame {
              background-color: #e8f5e9;
              border-radius: 12px;
              padding: 15px;
          }
         </string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayoutStock">
         <item>
          <widget class="QLabel" name="labelStock">
           <property name="styleSheet">
            <string>
             font-size: 18px;
             font-weight: bold;
             color: #2e7d32;
             margin-bottom: 10px;
            </string>
           </property>
           <property name="text">
            <string>üßº Stock</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayoutStock">
           <item row="0" column="0">
            <widget class="QLabel" name="labelStockLevel">
             <property name="text">
              <string>Stock Total :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLCDNumber" name="lcdStockLevel">
             <property name="styleSheet">
              <string>
               background-color: #2e7d32;
               color: white;
               border-radius: 6px;
              </string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="Employes">
        <property name="styleSheet">
         <string>
          QFrame {
              background-color: #e8f5e9;
              border-radius: 12px;
              padding: 15px;
          }
         </string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayoutEmployes">
         <item>
          <widget class="QLabel" name="labelEmployes">
           <property name="styleSheet">
            <string>
             font-size: 18px;
             font-weight: bold;
             color: #2e7d32;
             margin-bottom: 10px;
            </string>
           </property>
           <property name="text">
            <string>üë• Employ√©s</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayoutEmployes">
           <item row="0" column="0">
            <widget class="QLabel" name="labelEmployeCount">
             <property name="text">
              <string>Total Employ√©s :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLCDNumber" name="lcdEmployeCount">
             <property name="styleSheet">
              <string>
               background-color: #2e7d32;
               color: white;
               border-radius: 6px;
              </string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="Clients">
        <property name="styleSheet">
         <string>
          QFrame {
              background-color: #fff3e0;
              border-radius: 12px;
              padding: 15px;
          }
         </string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayoutClients">
         <item>
          <widget class="QLabel" name="labelClients">
           <property name="styleSheet">
            <string>
             font-size: 18px;
             font-weight: bold;
             color: #f57c00;
             margin-bottom: 10px;
            </string>
           </property>
           <property name="text">
            <string>üë®‚Äçüç≥ Clients</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayoutClients">
           <item row="0" column="0">
            <widget class="QLabel" name="labelClientCount">
             <property name="text">
              <string>Total Clients :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLCDNumber" name="lcdClientCount">
             <property name="styleSheet">
              <string>
               background-color: #f57c00;
               color: white;
               border-radius: 6px;
              </string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <!-- Plats pr√©par√©s -->
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutPlat">
      <item>
       <widget class="QFrame" name="Plats_prepares">
        <property name="styleSheet">
         <string>
          QFrame {
              background-color: #fff3e0;
              border-radius: 12px;
              padding: 15px;
          }
         </string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayoutPlats">
         <item>
          <widget class="QLabel" name="labelPlatsPrepares">
           <property name="styleSheet">
            <string>
             font-size: 18px;
             font-weight: bold;
             color: #f57c00;
             margin-bottom: 10px;
            </string>
           </property>
           <property name="text">
            <string>üç≥ Plats Pr√©par√©s</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayoutPlats">
           <item row="0" column="0">
            <widget class="QLabel" name="labelPlatsPrepares">
             <property name="text">
              <string>Plats Pr√©par√©s :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLCDNumber" name="lcdPlatsPrepares">
             <property name="styleSheet">
              <string>
               background-color: #f57c00;
               color: white;
               border-radius: 6px;
              </string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <!-- Bouton SORTIE -->
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="pushButtonExit">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string>
          QPushButton {
              background-color: #d32f2f;
              color: white;
              border-radius: 8px;
              padding: 10px;
          }
          QPushButton:hover {
              background-color: #b71c1c;
          }
         </string>
        </property>
        <property name="text">
         <string>üö™ SORTIE</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>

